src/server/services/VideoresServices.js

```diff
@@ -52,11 +53,19 @@ class VideoresServices {
     return docs;
   }
 
-  static async getAllVideoReserves(term) {
-    const termMedia = await MediaQuery.getMediaForTerm('videoreserves', term);
+  static async getAllVideoReserves() {
+    const termMedia = await MediaQuery.getMediaGroupedByShortname(
+      'videoreserves'
+    );
     return termMedia;
   }
 
+  static async getTerms() {
+    const terms = await MediaQuery.getTerms('videoreserves');
+    terms.sort(compareAcademicTerms).reverse();
+    return terms;
+  }
+
```

Comment:
Seems like Bruincast services, video res services, and music res services all use the same functions. Can we put it inside media resource services and reuse for all 3 cases?